.gitignore

```diff
@@ -1,4 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+#firebase
+.firebase
```

Comment:
Why are you ignoring this file?

Reply:
Since it contains cache files that are created during deployment, didn't find it necessary to add and push